File: server/dto/EndRoundDto.js
export class EndRoundDto
{
    player1EndRound;
    player2EndRound;
    playerIdOnBottom;
    
    constructor(player1EndRound, player2EndRound, playerIdOnBottom){
        this.player1EndRound = player1EndRound;
        this.player2EndRound = player2EndRound;
        this.playerIdOnBottom = playerIdOnBottom;
    }

    writeToBuffer(buffer)
    {
       this.player1EndRound.writeToBuffer(buffer);
       this.player2EndRound.writeToBuffer(buffer);
       
       let length = this.playerIdOnBottom.length;
       buffer.writeUInt8(length);
       buffer.writeString(this.playerIdOnBottom);
    }
}

File: Assets/Scripts/Client/dto/EndRoundDto.cs
namespace client.dto
{
    public class EndRoundDto : IDto
    {
        public PlayerEndRoundDto Player1EndRound { get; private set; }
        public PlayerEndRoundDto Player2EndRound { get; private set; }
        public string PlayerIdOnBottom { get; private set; }
            
        public void WriteToStream(CustomMemoryStream ms)
        {
            
        }

        public void ReadFromStream(CustomMemoryStream ms)
        {
            Player1EndRound = new PlayerEndRoundDto(ms);
            Player2EndRound = new PlayerEndRoundDto(ms);
            PlayerIdOnBottom = ms.ReadString();
        }
    }
}